contentpacks/models.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, *args, **kwargs):
 
 
 class AssessmentItem(Model):
-    id = CharField(max_length=50, primary_key=True)
+    id = CharField(max_length=50, index=True)
+    # looks like peewee doesn't like a primary key field that's not an integer.
+    # Hence, we have a separate field for the primary key.
+    pk = PrimaryKeyField(primary_key=True)
```

Comment:
Apparently it's already there: https://github.com/learningequality/ka-lite/blob/0.16.x/kalite/topic_tools/content_models.py#L46